pyarr/base.py

```diff
@@ -0,0 +1,627 @@
+from datetime import datetime
+
+from .request_api import RequestAPI
+
+
+class BaseAPI(RequestAPI):
```

Comment:
Awkward class naming.

Either rename `BaseAPI` -> `ArrAPI` or `BaseArrAPI`
... or rename `RequestAPI` -> `RequestHandler` or `RequestRouter`

I personally would suggest `RequestAPI` -> `RequestHandler`

Reply:
Agree updated this to be clear